src/main/webapp/js/chat.js

```diff
@@ -13,7 +13,7 @@ function createNewConversation(){
   }).then(() => {
     nicknameDom.value = '';
     inviteesDom.value = '';
-    buildConversations();
+    setTimeout( buildConversations(), 500);
```

Comment:
I also would've load conversations this way. But I wonder if you could load conversations only when new conversations are made. I mean, could you load the messages only when the fetched data's count is greater than the alr loaded data's count?

Reply:
Absolutely. I'm planning to add a time interval parameter in /chat/get/messages/ or Datastore cursor.